dbsync.py

```diff
@@ -284,13 +217,21 @@ def create_mergin_client():
         raise DbSyncError("Mergin Maps client error: " + str(e))
 
 
-def dbsync_pull(mc):
+def pull(conn_cfg, mc):
     """ Downloads any changes from Mergin Maps and applies them to the database """
 
-    _check_has_working_dir()
-    _check_has_sync_file()
+    print(f"Processing Mergin Maps project '{conn_cfg.mergin_project}'")
+    ignored_tables = ";".join(conn_cfg.skip_tables) if "skip_tables" in conn_cfg else ""
```

Comment:
I noticed this multiple times, I suggest to create a function in config for that, it is kind of parsing/reformatting config